src/Tribe/Editor/Compatibility.php

```diff
@@ -140,7 +140,7 @@ public function insert_toggle_blocks_editor_field( $fields = array() ) {
 			),
 		);
 
-		return Tribe__Main::array_insert_after_key( $insert_after, $fields, $insert_data );
+		return Tribe__Main::array_insert_before_key( $insert_before, $fields, $insert_data );
```

Comment:
Not required now, but sharing the mental note here: we should move this to the `Tribe__Utils__Array` class, redirect to that class and put a deprecated notice to note it.